docs/contributing/index.md

```diff
@@ -7,6 +7,7 @@ title: Contributing
 
 There are many ways that you can contribute to the LMS project with contributions of your time and effort:
 
+- [Engage in our community on our :simple-vbulletin: forums](https://forums.slimdevices.com/)
```

Comment:
Please remove the vBulletin icon here. Users will understand "forums", but have no idea what vBulletin is.

This actually would apply to the "social" buttons at the bottom too. But that's a nice to have for those who understand. The text here really is for everybody.

Reply:
Done